### PR TITLE
Add feature-flags library

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

This uses the API @chrisnojima provided in the ticket, and @MarcoPolo gave me a hand with flow checking for possible values.

```js
import featureFlags from 'util/feature-flags'
if (featureFlags.get('tracker') == 'v2') {
  ...
}
```

Next steps are to get flow checking for get/set if possible, and move to the store.  But the story asked to keep this as simple as possible (1pt) for now.

### CODE REVIEW

File: shared/util/feature-flags.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+type FeatureFlag = {
+  tracker: 'v1' | 'v2'
+}
+let ff: FeatureFlag = {
```
Comment (chrisnojima) [inline shared/util/feature-flags.js]: any reason why this isn't a static member?

Comment (chrisnojima) [inline shared/util/feature-flags.js]: also we want v2 to only be for us right, so we need some kind of example of this. either process.env or from local-debug or something

**Comment by MarcoPolo** [comment]
:+1: for now

here's another thought, why not make featureFlags a static value?

so then you would just do:

```js
import featureFlags from 'util/feature-flags'
if (featureFlags.tracker == 'v2') {
  ...
}
```

This also makes it easier to move it to the store, since you won't really want a dynamic object in the store just pure data.

Bonus: makes it trivial to type!

**Comment by cjb** [comment]
@chrisnojima @MarcoPolo Ready for another look!  Uses a static member and process.env.

File: shared/util/feature-flags.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* @flow */
+
+import {trackerVersionTwo} from '../local-debug'
```
Comment (chrisnojima) [inline shared/util/feature-flags.js]: sorry for the PR churn, i think its simpler to just have one or the other, either process.env or local-debug, i don't think we gain much by having both. so i'd just keep this file and just do 

```
tracker: process.env. KEYBASE_TRACKER_V2 ? 'v2' : 'v1'
```

**Comment by cjb** [comment]
Cool, accessing process.env directly now, merging in.

**Pull request merged**